src/renderer/component/fileViewer/internal/player.jsx

```diff
@@ -274,6 +293,8 @@ class MediaPlayer extends React.PureComponent {
       // Show unsupported error message
     } else if (isUnsupported || isUnplayable) {
       loader.loadingStatus = isUnsupported ? unsupportedMessage : unplayableMessage;
+    } else if (isLbryPackage && !isLoadingFile) {
+      loader.loadingStatus = false;
```

Comment:
Nevermind, I see it's used down here. It just seems kinda weird having very similar functionality.  Maybe `isLbryPackage` can be moved to its own function?

Reply:
If you have logic written out for this, I'm all ears.
See the original line 263, I'm doing this to avoid breaking existing logic.